apps/store/src/components/PriceCalculator/Warning/Warning.tsx

```diff
@@ -0,0 +1,47 @@
+import styled from '@emotion/styled'
+import { useState } from 'react'
+import { Dialog, theme } from 'ui'
+import { PriceIntent } from '@/services/priceIntent/priceIntent.types'
+import { Features } from '@/utils/Features'
+import { WarningPrompt } from './WarningPrompt/WarningPrompt'
+
+type Props = {
+  priceIntent: PriceIntent
```

Comment:
Should we change the props so it receives only `PriceIntentWarning`? After all it doesn't care about other `priceIntent` fields 🤔 

```suggestion
  priceIntent: PriceIntentWarning
```

Another suggestion would be bring whole implementation of `WarningPrompt` into this file and remove `WarningPrompt` all together. It's implementation (UI) is fairly simple so this file wouldn't grow too big.

Reply:
That makes sense, I'll see where we end up with the logic for when to display the Dialog. If it's simple enough we could put it in one file 👍 Otherwise I think it's nice to keep the same pattern as the FetchInsurancePrompt